meta-lmp-support/files/pelion-group-table

```diff
@@ -0,0 +1,8 @@
+parsec:x:1004:
```

Comment:
```suggestion
parsec:x:1206:
```

Reply:
The current values used (in 2.4.0) are 1206, and I think these need to be used going forwards. An ostree update won't take the new values into use, due to the 3 way merge functionality.